README.rst

```diff
@@ -61,8 +61,7 @@ Usage
 
   import itk
 
-  imageio = itk.ScancoImageIO.New()
-  image = itk.imread('myvolume.ISQ', imageio=imageio)
+  image = itk.imread('myvolume.ISQ')
```

Comment:
Which change brought this improvement?

Reply:
itk-5.3.0 (also present in recent RC's).

Reply:
I meant which PR in ITK, or commit SHA is responsible. Just curious.

Reply:
https://github.com/InsightSoftwareConsortium/ITK/pull/3156